config/roles.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    "roles" => [
+        ["name" => "super-admin", "guard_name" => "web"],
```

Comment:
```suggestion
        ["name" => "super_admin", "guard_name" => "web"],
```

remove guard key, I think.